app/models/item.rb

```diff
@@ -1,7 +1,11 @@
 class Item < ApplicationRecord
-    belongs_to :merchant
-    has_many :invoice_items
-    has_many :invoices, through: :invoice_items
-    has_many :customers, through: :invoices
-    has_many :transactions, through: :invoices
+  belongs_to :merchant
+  has_many :invoice_items
+  has_many :invoices, through: :invoice_items
+  has_many :customers, through: :invoices
+  has_many :transactions, through: :invoices
+
+  def to_dollars
+    "$" + sprintf('%.2f', self.unit_price.to_f / 100)
```

Comment:
nice! I used this for dates formatting. Cool to see it work this way too.

Reply:
FYI Noah asked about this in codehelp. I'll ping on slack so this comment doesn't get lost. Abdul said rails views have a `number_to_currency` method to use for this case. Kat agreed that this might make sense to be in the view not model, since it's just changing the view. I don't know if that's suggested or required though.